modules/web/src/app/wizard/component.ts

```diff
@@ -176,9 +193,49 @@ export class WizardComponent implements OnInit, OnDestroy {
       .afterClosed()
       .pipe(filter(ct => !!ct))
       .pipe(take(1))
-      .subscribe(ct =>
-        this._notificationService.success(`${isEditMode ? 'Saved' : 'Updated'} the ${ct.name} cluster template`)
-      );
+      .subscribe(ct => {
+        this._notificationService.success(`${isEditMode ? 'Updated' : 'Saved'} the ${ct.name} cluster template`);
+        this._router.navigate([`/projects/${this.project.id}/clustertemplates`], {
+          fragment: saveAndCreate && `${ct.name}`,
+        });
```

Comment:
We should add a check here and use router `state` instead of `fragment` so its not visible in the URL.
```suggestion
       if (this.wizardMode === WizardMode.CreateClusterTemplate || (this.wizardMode === WizardMode.EditClusterTemplate && !isNew)) {
          this._router.navigate([`/projects/${this.project.id}/clustertemplates`], {
            state: createCluster ? {
              createClusterFromTemplateID: ct.id
            } : null
          });
        }
```